### ISSUE TITLE
Tagging and Tag Filters (New Feature)

### ISSUE THREAD

**Issue by rickcecil** [Zuntek/WeekToDoWeb#70]
Loving WeekToDo. I believe that it fills a void in the productivity space. Haven't quite seen anything like it. 

One thing that would really elevate, I think, is tags. Just adding #tag at the end of a todo item and then being able to 

- view all the todo items associated with a given tag
- Filter the calendar view by tag

I'll try to post some concepts for these things in the next few days.

**Comment by merodriguez9112** [comment]
Thanks for writing @rickcecil. Sounds like a good feature. I will add it to the list of features for future versions. :)

**Comment by MattDemers** [comment]
Just +1ing this, mostly to give over-arching projects some organization, if possible.

**Comment by MattDemers** [comment]
Yup, can totally understand that desire for simplicity. I don't want to force your project beyond its scope; part of why I started using it was because I was overloading myself with busywork from Asana.

**Comment by rickyspears** [comment]
For a long time I've though that many applications could benefit from a minimal set of features enabled out-of-the-box. Other features could then be enabled as needed. It could help keep the user interface uncluttered for almost everyone. And for our less technical users, it would help greatly with on-boarding — train them to use the basic features and then they can enable other features as they need them.

**Comment by manuelernestog** [comment]
100% agreed @rickyspears 👏

**Comment by SaifEl-Deen** [comment]
Is this going to be on v2.1?

**Comment by manuelernestog** [comment]
Nop

**Comment by SaifEl-Deen** [comment]
:crying in the corner:

**Comment by manuelernestog** [comment]
Let's move on to this to the v2.2, this needs some work and it would delay the release of v2.1

**Comment by SaifEl-Deen** [comment]
@manuelernestog Nice! Much appreciated that it is planned to 2.2 🤍

**Comment by SaifEl-Deen** [comment]
@manuelernestog Just a little request, is it possible that at least we get a basic tag feature in v2.1 where if you type # + a word, it will turn into text with some light blue background?
Just that, no custom color no other customizations.

**Comment by manuelernestog** [comment]
> @manuelernestog Just a little request, is it possible that at least we get a basic tag feature in v2.1 where if you type # + a word, it will turn into text with some light blue background? Just that, no custom color no other customizations.

Sorry man, that's not going to be possible, that would imply unnecessary work and moreover introduce a feature that would conflict with the actual tag system

**Comment by SaifEl-Deen** [comment]
I see, no problem, I'll be waiting for v2.2 then, hope the whole development process goes well and free of issues.

**Comment by lucaszischka** [comment]
Hey @manuelernestog, just found this thread and im using v.2.2. Did you not add the tagging feature or do I have to enable it? Thank you!

**Comment by nomad101** [comment]
This is a great feature and will make WeekToDo even more useful. Though as with the previous poster, I couldn't find it in V.2.2? Thanks
